app/gui2/src/components/GraphEditor/widgets/WidgetVector.vue

```diff
@@ -55,7 +55,7 @@ export const widgetDefinition = defineWidget(WidgetInput.isAstOrPlaceholder, {
     else if (props.input.expectedType?.startsWith('Standard.Base.Data.Vector.Vector'))
       return Score.Good
     else if (props.input.value instanceof Ast.Ast)
-      return props.input.value.treeType === RawAst.Tree.Type.Array ? Score.Perfect : Score.Mismatch
+      return props.input.value.children().next().value.code === '[' ? Score.Perfect : Score.Mismatch
```

Comment:
This is temporary. I'm eliminating `RawAst` type access in favor of `instanceof` checks, but I haven't written `Ast.Vector` yet (it will be as complex as `BodyBlock` due to the need to manage concrete tokens).